rules/port-forwarding/rule.md

```diff
@@ -39,36 +39,7 @@ Luckily, port forwarding is the silver...errr...golden bullet that solves this i
 
 ## Dev Tunnels Setup
```

Comment:
It feels like we're not just sending them to the docs but rather deleting the whole section. Either keep all or lose all